OWNERS_ALIASES

```diff
@@ -160,9 +160,9 @@ aliases:
   committee-code-of-conduct:
     - AnaMMedina21
     - endocrimes
-    - hlipsig
     - jeremyrickard
-    - salaxander
+    - divya-mohan0209
+    - stmcginnis
```

Comment:
Ordering nit for the current committee names. My name should probably be in between Ana's & your names.